starter-dev-backend/routes/index.js

```diff
@@ -0,0 +1,7 @@
+const routes = require('express').Router();
+const { graphqlHTTP } = require('express-graphql');
+const graphql = require('./graphql');
+
+routes.use('/graphql', graphql);
```

Comment:
```suggestion
routes.use('/', graphql);
```

Per the above suggestion